Makefile

```diff
@@ -265,12 +265,15 @@ ci-secret-bootstrap:
 
 ci-secret-generator:
 	$(CONTAINER_ENGINE) pull registry.svc.ci.openshift.org/ci/ci-secret-generator:latest
-	$(CONTAINER_ENGINE) run --rm -v "$(CURDIR)/core-services/ci-secret-generator/_config.yaml:/_config.yaml:z" -v "$(CURDIR)/core-services/ci-secret-bootstrap/_config.yaml:/_bootstrap_config.yaml:z" \
-		-v "$(kubeconfig_path):/_kubeconfig:z" \
-		-v "$(bw_password_path):/_bw_password:z" \
-		-e KUBECONFIG=_kubeconfig \
-		registry.svc.ci.openshift.org/ci/ci-secret-generator:latest \
-		--bw-password-path=/_bw_password --bw-user $(kerberos_id)@redhat.com --config=/_config.yaml --bootstrap-config=/_bootstrap_config.yaml --dry-run=$(dry_run)
+	@# This needs a bunch of stuff from the host for auth so just copy it there
+	$(eval ID = $(shell docker create registry.svc.ci.openshift.org/ci/ci-secret-generator))
```

Comment:
s/docker/$(CONTAINER_ENGINE)/

Reply:
I personally find that just doing an `alias docker=YOUR_ENGINE`  would be easier than adjusting scripts AND requiring ppl to still export an env var telling the script what container engine to use

Reply:
Then fixup the whole thing, but it should be consistent

Reply:
done

Reply:
oh, wow

I guess I asked for it :D

Reply:
meh.

`alias` won't work for me, because Make is spinning in its own shell. Docker won't work for me out-of-the-box, because I need to override the `REGISTRY_AUTH_FILE` which Docker doesn't let me as easy as Podman.

@alvaroaleman  I am not sure why you needed this change: if your docker alias works for you now, it most probably worked even with `CONTAINER_ENGINE` set to `docker` (the default).

As a side note, I wonder why we default to Docker in RH.